tests/test_model_helpers/activations/test___init__.py

```diff
@@ -189,6 +189,58 @@ def test_from_image_path(model_ctr, layers, image_name, pca_components, logits):
     return activations
 
 
+@pytest.mark.parametrize("image_name", ['rgb.jpg', 'grayscale.png', 'grayscale2.jpg', 'grayscale_alpha.png',
+                                        'palletized.png'])
+@pytest.mark.parametrize(["model_ctr", "layers"], models_layers)
+@pytest.mark.parametrize("number_of_trials", [1, 5, 25])
+def test_microsaccades_from_image_path(model_ctr, layers, image_name, number_of_trials):
+    stimulus_paths = [os.path.join(os.path.dirname(__file__), image_name)]
+    activations_extractor = model_ctr()
+
+    activations = activations_extractor(stimuli=stimulus_paths, layers=layers, number_of_trials=number_of_trials,
+                                        require_variance=True)
+
+    assert activations is not None
+    assert len(activations['shift_x']) == number_of_trials * len(stimulus_paths)
+    assert len(activations['shift_y']) == number_of_trials * len(stimulus_paths)
+
+
+@pytest.mark.parametrize("image_name", ['rgb.jpg', 'grayscale.png', 'grayscale2.jpg', 'grayscale_alpha.png',
+                                        'palletized.png'])
+@pytest.mark.parametrize(["model_ctr", "layers"], models_layers)
+@pytest.mark.parametrize("require_variance", [False, True])
+def test_model_requirements(model_ctr, layers, image_name, require_variance):
+    stimulus_paths = [os.path.join(os.path.dirname(__file__), image_name)]
+    activations_extractor = model_ctr()
+
+    activations_with_req = activations_extractor(stimuli=stimulus_paths, layers=layers,
+                                                 number_of_trials=2, require_variance=require_variance)
+    activations_without_req = activations_extractor(stimuli=stimulus_paths, layers=layers,
+                                                    number_of_trials=1, require_variance=False)
```

Comment:
confused why we parametrize by `require_variance` but then also have to `activations_extractor` calls